charts/cluster-autoscaler/README.md

```diff
@@ -81,10 +81,16 @@ Auto-discovery finds ASGs tags as below and automatically manages them based on
 - Verify the [IAM Permissions](#aws---iam)
 - Set `autoDiscovery.clusterName=<YOUR CLUSTER NAME>`
 - Set `awsRegion=<YOUR AWS REGION>`
-- Set `awsAccessKeyID=<YOUR AWS KEY ID>` and `awsSecretAccessKey=<YOUR AWS SECRET KEY>` if you want to [use AWS credentials directly instead of an instance role](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#using-aws-credentials)
+- Set (option) `awsAccessKeyID=<YOUR AWS KEY ID>` and `awsSecretAccessKey=<YOUR AWS SECRET KEY>` if you want to [use AWS credentials directly instead of an instance role](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#using-aws-credentials)
 
 ```console
-$ helm install my-release autoscaler/cluster-autoscaler --set autoDiscovery.clusterName=<CLUSTER NAME>
+$ helm install my-release autoscaler/cluster-autoscaler --set autoDiscovery.clusterName=<CLUSTER NAME> --set awsRegion=<YOUR AWS REGION>
+```
+
+Alternatively with your own AWS credencials
```

Comment:
Thanks, the changes looks useful to me (I'm not a maintainer though), just noticed a small nit:
```suggestion
Alternatively with your own AWS credentials
```